db/schema.rb

```diff
@@ -242,6 +249,13 @@
     t.index ["user_id"], name: "index_courses_users_on_user_id"
   end
 
+  create_table "courses_wikis", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.integer "course_id"
+    t.integer "wiki_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
To make the common queries as fast as possible, this should be indexed by course, and probably by wiki as well.

Reply:
This should also include a uniqueness constraint: we don't want duplicate CoursesWikis records for the same course+wiki combination.